Scripts/mssql-performance-statistics-bind.sql

```diff
@@ -1,14 +1,12 @@
 -- Execute query to see the performance statistics using bind variables.
 
-USE [LANSA]
```

Comment:
@robe070 - **Use [Lensa]** line removed in the .sql file and added the database name in the **SQLCMD.exe** script in PowerShell script.  so the database is connected via sqlcmd.exe and it is not required in the script file. This a way to remove suppression of DB context messages in the command line. 

Please tell me the reason to required this line so that we can consider and find an alternative for that.

Reply:
> @robe070 - **Use [Lensa]** line removed in the .sql file and added the database name in the **SQLCMD.exe** script in PowerShell script. so the database is connected via sqlcmd.exe and it is not required in the script file. This a way to remove suppression of DB context messages in the command line.
> 
> Please tell me the reason to required this line so that we can consider and find an alternative for that.

I think what you are asking is "is your solution of passing it on the SQLCMD line valid?" It is. Use LANSA is not required.